File: 2021/08/1.rs
use std::fs::File;
use std::io::{self, BufRead};

fn main() {
    let file = File::open("input").unwrap();
    let sum = io::BufReader::new(file).lines().map(|x| x.unwrap()).map(|line|
        line.split(' ').skip(11).filter(|x| vec![2, 3, 4, 7].contains(&x.len())).count()
    ).sum::<usize>();
    println!("{}", sum);
}

File: 2023/05/1.py
lines = open("input", "r").read().split("\n")
seeds = [int(x) for x in lines[0].split()[1:]]
new_seeds = list(seeds)
for l in lines[2:]:
    if 'map' in l:
        seeds, new_seeds = new_seeds, list(seeds)
        continue
    if l:
        dr, sr, ln = [int(x) for x in l.split()]
        for i, s in enumerate(seeds):
            if sr <= s < sr + ln:
                new_seeds[i] = s - sr + dr
print(min(new_seeds))
